app/api/routers/route_factory.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import Request
+
+from .. import crud
+
+
+def create_get_instances_endpoint(
```

Comment:
I think a better name is 
```suggestion
def create_get_instances_handler(
```
because the endpoint is already defined, you're supplying the handler function here.